src/Serialization/main.jl

```diff
@@ -240,11 +240,17 @@ function save(filename::String, obj::Any)
 end
 
 """
-    load(io::IO)
-    load(filename::String)
+    load(io::IO; parent::Any = nothing, T::Type = nothing)
+    load(filename::String; parent::Any = nothing, T::Type =  nothing)
 
 Load the object stored in the given io stream
 respectively in the file `filename`.
+The optional parameter `parent` can be passed. This will
+load the file as an element of the parent, or for container types
+such as `Vector` and `Tuple` it will load the entries as elements of the
+passed `parent`.
```

Comment:
That a parameter can be passed seems rather self-evident ;-). Also, the source
may not be a file. Perhaps:
```suggestion

If `parent` is specified, then the root object of the loaded data
either will have this as its parent, or it is a container such as
`Vector` and `Tuple`, then the parent of the entries will be set to `parent`.
```

Reply:
yikes, sorry, that's embarrassing, thanks!